modules/dr-hosted-cluster-within-aws-region-delete.adoc

```diff
@@ -87,41 +106,83 @@ done
 
 . Delete the `HostedControlPlane` and `ControlPlane` HC namespace objects by entering these commands:
 +
+.Delete HCP and ControlPlane HC NS commands
 [source,terminal]
 ----
-# Delete HCP and ControlPlane HC NS
 $ oc patch -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} hostedcontrolplane.hypershift.openshift.io ${HC_CLUSTER_NAME} --type=json --patch='[ { "op":"remove", "path": "/metadata/finalizers" }]'
+----
++
+[source,terminal]
+----
 $ oc delete hostedcontrolplane.hypershift.openshift.io -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} --all
+----
++
+[source,terminal]
+----
 $ oc delete ns ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} || true
 ----
 
 . Delete the `HostedCluster` and HC namespace objects by entering these commands:
 +
+.Delete HC and HC Namespace commands
 [source,terminal]
 ----
-# Delete HC and HC Namespace
 $ oc -n ${HC_CLUSTER_NS} patch hostedclusters ${HC_CLUSTER_NAME} -p '{"metadata":{"finalizers":null}}' --type merge || true
+----
++
+[source,terminal]
+----
 $ oc delete hc -n ${HC_CLUSTER_NS} ${HC_CLUSTER_NAME}  || true
+----
++
```

Comment:
```suggestion
+.Delete Namespace commands
```
if you are going to break up commands, you should define each, not leave a series unlabeled